app/src/main/kotlin/ch/hsr/ifs/gcs/ui/MainActivity.kt

```diff
@@ -211,8 +221,14 @@ class MainActivity : AppCompatActivity(), ActivityCompat.OnRequestPermissionsRes
         map.controller.setZoom(18.0)
         map.setBuiltInZoomControls(true)
 
-
         MyLocationNewOverlay(fLocationProvider, map).apply {
```

Comment:
Have you considered using `setPersonHotspot(float, float)` to prevent the "bouncing" of the icon?

Reply:
Added `setPersonHotspot(69.5f, 69.5f)`. This seems to fix the issue. However, I have no idea how this could be calculated.